s2i-liberty/readme.md

```diff
@@ -0,0 +1,197 @@
+
+# Liberty Source to Image Builder Demo
+
+This demonstration describes how to produce a new Source to Image (S2I) runtime image to deploy web application running on liberty on OpenShift.
+
+![IBM Liberty](https://browser-call.wasdev.developer.ibm.com/assets/liberty_logo_transp.png "IBM Liberty")
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Bill of Materials](#bill-of-materials)
+	* [Environment Specifications](#environment-specifications)
+	* [Template Files](#template-files)
+	* [Config Files](#config-files)
+	* [External Source Code Repositories](#external-source-code-repositories)
+* [Setup Instructions](#setup-instructions)
+* [Presenter Notes](#presenter-notes)
+	* [Environment Setup](#environment-setup)
+	* [Produce Runtime Image](#produce-runtime-image)
+		* [Git Source](#git-source)
+		* [Binary Source](#binary-source)
+	* [Create the build config](#create-the-build-config)	
+	* [Create A New Application](#create-a-new-application)
+    * [Considerations on HTTP session failover](#considerations-on-http-session-failover)	
+
+
+## Overview
+
+OpenShift provides several out of the box Source to Image builder images. To support deployments in Liberty, a new s2i builder image will be created to support a simplified deployment to OpenShift. Once the new image is produced, an example application will be deployed. 
+
+We will use the [extended builds approach](https://docs.openshift.org/latest/dev_guide/builds.html#extended-builds).
```

Comment:
This should reference the OCP docs instead of origin

https://docs.openshift.com/container-platform/3.3/dev_guide/builds.html#extended-builds